dotCMS/src/main/java/com/dotcms/rendering/velocity/services/ContainerLoader.java

```diff
@@ -149,6 +155,10 @@ private InputStream buildVelocity(Container container, String uuid, PageMode mod
                     .append(" data-max-contentlets=")
                     .append("\"" + container.getMaxContentlets() + "\"")
                     .append(" data-dot-accept-types=")
+                    .append(" data-dot-can-edit=\"")
+                    .append(String.valueOf(APILocator.getPermissionAPI().doesUserHavePermission(container,
+                            PermissionLevel.WRITE.getType(), user,true)))
```

Comment:
Yeah, there will be no user here. Or it will cache the user that first called it.

Reply:
also, this broke the data-dot-accept-types, which means you cannot drag anything anywhere.

Reply:
Yes wrong place to put the new data-dot-can-edit